src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("the excavator started it`s work");
+
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("the excavator stopped it`s work");
```

Comment:
Use proper apostrophes in `it's` instead of backticks. The correct punctuation for contraction of 'it is' is `it's`.